packages/pds/src/index.ts

```diff
@@ -45,6 +45,7 @@ import { Crawlers } from './crawlers'
 import { LabelCache } from './label-cache'
 import { ContentReporter } from './content-reporter'
 import { ModerationService } from './services/moderation'
+import { AtpAgent } from '@atproto/api'
```

Comment:
Tiny baby nit: could be nice to hoist this up near the other `@atproto/*` deps.

Reply:
yup yup 👍